### PR TITLE
Refactor/#65/refactor authors service

### PR DESCRIPTION
**Pull request by Bogdan-Zinovij**
Refactored authors servise, added validation, removed unnecessary endpoint, fix .eslintrc filename

### CODE REVIEW

File: services/authors/src/services/author.service.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+'use strict';
```
Comment (KhrapkoVasyl) [inline services/authors/src/services/author.service.js]: extra 'use strict'

**Review by KhrapkoVasyl — APPROVED** [review]
Remove extra 'use strict' and merge

**Review by DanilByaliy — APPROVED** [review]

File: services/authors/src/services/author.service.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    if (!author) throw new Error(errorMessages.AUTHOR_NOT_EXISTS);
+
+    return author;
+  }
+
+  async createAuthor(authorData) {
+    const author = await Authors.findOne({ where: { name: authorData.name } });
+    if (author) throw new Error(errorMessages.AUTHOR_ALREADY_EXISTS);
+
+    const newAuthor = { ...authorData };
```
Comment (artemmatiushenko1) [inline services/authors/src/services/author.service.js]: Can be written in one line 
```js
const newAuthor = { ...authorData, id: uuid()}
```

**Review by artemmatiushenko1 — CHANGES_REQUESTED** [review]

File: services/authors/src/services/author.service.js (hunk 2 above)
Comment (artemmatiushenko1) [inline services/authors/src/services/author.service.js]:
```suggestion
    const newAuthor = { ...authorData, id: uuid() };
```

File: services/authors/src/services/author.service.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    newAuthor.id = uuid();
```
Comment (artemmatiushenko1) [inline services/authors/src/services/author.service.js]:
```suggestion

```

**Review by artemmatiushenko1 — APPROVED** [review]

**Pull request merged**